mono/profiler/log-args.c

```diff
@@ -132,8 +132,6 @@ parse_arg (const char *arg, ProfilerConfig *config)
 		if (config->cov_filter_files == NULL)
 			config->cov_filter_files = g_ptr_array_new ();
 		g_ptr_array_add (config->cov_filter_files, g_strdup (val));
-	} else if (match_option (arg, "onlycoverage", NULL)) {
```

Comment:
Well, I'm removing it because it has no meaning after this change. It was only there to filter out enter/leave events when doing coverage, but we don't even enable those events for coverage anymore.